src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerConfigTests.cs

```diff
@@ -2492,6 +2492,16 @@ public void InvalidGlobalLevelIsIgnored()
         [InlineData("/dir1/dir3/dir4/../dir2/file.cs", false)]
         [InlineData("file.cs", false)]
         [InlineData("", false)]
+        [InlineData("/../../dir1/dir2/file.cs", true)]
+        [InlineData("/./../dir1/dir2/file.cs", true)]
+        [InlineData("/dir1/../../dir1/dir2/file.cs", true)]
+        [InlineData("/..", false)]
+        [InlineData("/../file.cs", false)]
+        [InlineData("/dir1/../file.cs", false)]
+        [InlineData("./dir1/dir2/file.cs", false)]
+        [InlineData("././../.././dir1/dir2/file.cs", false)]
+        [InlineData("./dir1/../file.cs", false)]
+        [InlineData("../dir1/dir2.cs", false)]
```

Comment:
Instead of 'shouldMatch', should we just include a 'string expected' in this data? For example, the expected result of normalizing `./dir1/../file.cs` is `file.cs`.

Reply:
I ended up doing that on the direct test, so we can get decent coverage.